omnigibson/simulator.py

```diff
@@ -332,13 +336,13 @@ def _set_physics_engine_settings(self):
 
         def _set_renderer_settings(self):
             # TODO: For now we are setting these to some reasonable high-performance values but these can be made configurable.
-            lazy.carb.settings.get_settings().set_bool("/rtx/reflections/enabled", False)  # Can be True with a 10fps penalty
-            lazy.carb.settings.get_settings().set_bool("/rtx/indirectDiffuse/enabled", True)  # Can be False with a 5fps gain
-            lazy.carb.settings.get_settings().set_bool("/rtx/directLighting/sampledLighting/enabled", True)
+            lazy.carb.settings.get_settings().set_bool("/rtx/reflections/enabled", False)                   # Can be True with a fps penalty
+            lazy.carb.settings.get_settings().set_bool("/rtx/indirectDiffuse/enabled", False)               # Can be True with a fps penalty
```

Comment:
one small thing here -- can we keep the performance-penalty settings here if gm.ENABLE_HQ_RENDERING=True?